addons/mail/static/src/components/composer/composer.scss

```diff
@@ -251,11 +227,16 @@
 
 .o_Composer_toolButton {
     border: 0;
+    color: gray('600');
 
     &:hover {
         // enforce this style (webclient btn-secondary)
         border: 0;
     }
+
+    &.o-open {
+        background-color: gray('200');
+    }
 }
```

Comment:
This whole class is not good either. `o_Composer_toolButton` is on an element with .btn .btn-secondary, we shouldn't try to randomly override some values, as this will cause issues with every style that we "missed" since it relies on assumptions about current values of "secondary" that are subject to change.

Maybe we can try to use a standard bootstrap style for this button, such as -link or -light, and if none works, we shouldn't use bootstrap classes at all and do everything by hand it (and this is not ideal to do right now because button style is quite complex).

Reply:
![image](https://user-images.githubusercontent.com/48204996/85539706-706dc800-b616-11ea-874a-e3440added5c.png)

Seems odd, the colors are not exactly alright with .btn-light instead of .btn-secondary

```
.o_Composer_toolButton {
    border: 0;
}
```

With .btn-link the buttons are too visible I think 
![image](https://user-images.githubusercontent.com/48204996/85540173-f5f17800-b616-11ea-94b4-ccfab83f4ebd.png)

and the hover effects are not that great ... (emoji button is hovered here)

![image](https://user-images.githubusercontent.com/48204996/85540229-09044800-b617-11ea-9482-895ed2c8cf9c.png)

Reply:
btn-link looks much better than grey for me when composer is compact, there is a nice harmony with the color of the button. But maybe that's not the intent of changing the spec here.

Light could be fine if it was tweaked just a little bit to be actually a lighter color, I think that's a fine override to make, at least much better than basing our override on secondary that is totally different.